test/jdk/java/awt/InputMethods/DiacriticsTest/DiacriticsTest.java

```diff
@@ -22,29 +22,94 @@
  */
 
 
+import jdk.test.lib.Platform;
```

Comment:
Additional line space before imports can be removed.

Reply:
Removed